app/views/race_editions/enter.html.erb

```diff
@@ -15,7 +15,7 @@
       <%= racer_fields.text_field :email %>
     
       <%= racer_fields.label :gender %>
-      <%= racer_fields.select :gender, Racer.genders.keys, {} %>
+      <%= racer_fields.select :gender, Racer.genders.keys, include_blank: true %>
```

Comment:
No gender will be auto-selected; entrants will have to affirmatively pick a gender. If the form is submitted without a gender chosen, validation will fail and the form will be re-rendered.